<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="deviceWholeView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>211</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="timerButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>⏲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="increaseButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decreaseButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>⏯️</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);
font: 75 9pt &quot;Noto Sans Adlam&quot;;</string>
    </property>
    <property name="text">
     <string>Alpha-Stim CES Device</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="powerOffView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0)
</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="userView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>241</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Adlam</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="title2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);
font: 75 9pt &quot;Noto Sans Adlam&quot;;</string>
    </property>
    <property name="text">
     <string>User</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="attachButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Attach</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detachButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Detach</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>⏻</string>
    </property>
   </widget>
   <widget class="QPushButton" name="waveFormButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>∿</string>
    </property>
   </widget>
   <widget class="QPushButton" name="frequencyButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>f</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 41, 41)</string>
    </property>
    <property name="text">
     <string>⬤</string>
    </property>
   </widget>
   <widget class="QPushButton" name="currentButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Adlam</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>	⚡</string>
    </property>
   </widget>
   <widget class="QLabel" name="batteryLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery: 100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="recordLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Recording: Off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>↵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>✓</string>
    </property>
   </widget>
   <zorder>deviceWholeView</zorder>
   <zorder>timerButton</zorder>
   <zorder>increaseButton</zorder>
   <zorder>decreaseButton</zorder>
   <zorder>startButton</zorder>
   <zorder>title</zorder>
   <zorder>userView</zorder>
   <zorder>title2</zorder>
   <zorder>attachButton</zorder>
   <zorder>detachButton</zorder>
   <zorder>powerButton</zorder>
   <zorder>waveFormButton</zorder>
   <zorder>frequencyButton</zorder>
   <zorder>recordButton</zorder>
   <zorder>currentButton</zorder>
   <zorder>powerOffView</zorder>
   <zorder>backButton</zorder>
   <zorder>checkButton</zorder>
   <zorder>recordLabel</zorder>
   <zorder>batteryLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
